packages/neuron-wallet/src/services/cells.ts

```diff
@@ -335,7 +335,7 @@ export default class CellsService {
         o.typeCodeHash === nftIssuerCodehash ||
         o.typeCodeHash === nftClassCodehash ||
         o.typeCodeHash === nftCodehash ||
-        (o.typeCodeHash != null && o.hasData)
+        (o.typeCodeHash !== null && o.hasData)
```

Comment:
I'm not sure if it is by design to include `o.typeCodeHash !== undefined`, can we use `!o.typeCodeHash` here? @yanguoyu

Reply:
It will save as `null` default https://github.com/Magickbase/neuron/blob/311488acd08fb7b6d0d4707af56f92fbc153ab37/packages/neuron-wallet/src/database/chain/entities/output.ts#L53. I think it will not be `undefined`.
`!==` or `!o.typeCodeHash` all will be ok.

Reply:
```suggestion
        (!o.typeCodeHash && o.hasData)
```